docs/product/dev-toolbar/setup.mdx

```diff
@@ -9,50 +9,58 @@ description: "Get started with Sentry's Dev Toolbar, bringing critical Sentry in
   [GitHub](https://github.com/getsentry/sentry-toolbar/issues) if you have any feedback or concerns.
 </Alert>
 
-## Set Up Your Web App
+## Pre-requisites
 
-<Alert level="warning">
-    [Enabling tracing](/platforms/javascript/tracing/) is a prerequisite to using the Dev Toolbar. Tracing is used to collect page-specific issues and feedback through transactions.
-</Alert>
+For the Sentry toolbar to work best [enable tracing](/platforms/javascript/tracing/) in your app. With tracing enabled the Dev Toolbar will be able to associate issues and feedback with the current url in the browser location.
 
-You need to complete two steps to get the toolbar rendered on the page:
-1. Add or dynamically inject the following script into your web app:
-    ```html
-    <script src="https://browser.sentry-cdn.com/sentry-toolbar/latest/toolbar.min.js"></script>
-    ```
-    It is recommended that you add the script tag at the bottom of the page so it doesn’t block other critical JavaScript.
-
-2. Call `window.SentryToolbar.init(initConfig)` to set up a toolbar instance on each page where you want to see the Dev Toolbar. Take care to also [conditionally load the toolbar in your production environment](#deploying-to-production-environments).
-    ```html
-    <html>
-    <head>...</head>
-    <body>
-        ...
-        <script src="https://browser.sentry-cdn.com/sentry-toolbar/latest/toolbar.min.js"></script>
-        <script>
-        window.SentryToolbar.init({ ... });
-        </script>
-    </body>
-    </html>
-    ```
-
-3. Edit [Project Settings](https://sentry.io/orgredirect/organizations/:orgslug/settings/projects/) in Sentry so the Toolbar is allowed to connect from your dev, staging, and production domains.
-  ![Sentry's Dev Toolbar Settings Page](./img/sentry-project-settings-toolbar.png)
+## Install
```

Comment:
The steps to set up the Dev Toolbar are implied when you say "**The first step** is to include the toolbar code in your app." and "Next you will need to edit..."
but I think for easier digestion of the onboarding it would be nice if the steps were enumerated in some way?

It's also unclear which steps are optional v. required (for example, enabling FF is _technically_ optional as an organization may not use them in their development)

Reply:
I'll put numerals before the first three steps:
![SCR-20250204-mkfb](https://github.com/user-attachments/assets/527ebac0-a1ec-4c95-b764-a46ddfdcf41a)

Reply:
updated again. see the description for the latest screenshot of the outline / right sidebar.